translations/blocks/es-ar.yaml

```
@@ -133,3 +133,17 @@ procedures:
   noParams: ""
   addParam: "Agregar parámetro"
   addParamPrompt: "Ingresa el nombre del parámetro"
+actions:
+  createProcedure: "Crear '%1'"
```

```suggestion
  createProcedure: "Usar procedimiento '%1'"
```

Al enseñar hablamos mucho de uso vs definición, y está bueno que eso esté en la plataforma.